bcipy/signal/model/offline_analysis.py

```diff
@@ -29,6 +31,10 @@
 log = logging.getLogger(__name__)
 logging.basicConfig(level=logging.INFO, format="[%(threadName)-9s][%(asctime)s][%(name)s][%(levelname)s]: %(message)s")
 
+DIPSIZE = (1707, 1067)
+IMG_PATH = f'{STATIC_IMAGES_PATH}/main/matrix_grid.png'
+# IMG_PATH = f'{STATIC_IMAGES_PATH}/main/rsvp.png'
```

Comment:
We might assert this only works with matrix grids.. we can address this later though

Reply:
Added a `TODO` statement to circle back!